src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A immutable interface to display a Peron's details
```

Comment:
not necessary to be only person's details, no?
any class can implement this.
your header comment should be precise!